Services/DeviceClient.cs

```diff
@@ -6,6 +6,7 @@
 using System.Threading.Tasks;
```

Comment:
Will also remove changes to this file.

Reply:
do you think I should wait for the final version before completing the CR?

Reply:
Updated -- removed changes. To be submitted with pr for https://github.com/Azure/device-simulation-dotnet/tree/send-twin-updates